frontend/src/features/admin-form/common/queries.ts

```diff
@@ -39,8 +40,8 @@ export const useAdminForm = (
     ReturnType<typeof adminFormKeys.id>
   >,
 ): UseQueryResult<AdminFormDto, ApiError> => {
-  const { formId } = useParams()
-  if (!formId) throw new Error('No formId provided')
+  const { formId } = useCollaboratorWizard()
+  if (!formId) throw new Error('No formId provided to useAdminForm')
```

Comment:
maybe instead of having implicit coupling like before, we can pass in formId as a prop to this hook:

```tsx
export const useAdminForm = ( 
  formId: string, // add as param here
  props?: UseQueryOptions<
    AdminFormDto,
    ApiError,
    AdminFormDto,
    ReturnType<typeof adminFormKeys.id>
  >,
): UseQueryResult<AdminFormDto, ApiError> => {
```

this means this hook does not need the precondition anymore